docs/guides/entry_points.md

```diff
@@ -0,0 +1,90 @@
+
+You also can easily imagine offering renting a car, or a valuable asset and needing a quote from your insurance company, based on many factors: information about the person renting, time and duration, etc. 
+
+Let‘s say you want to test asking for fast and cheap options. Or you could be testing a new provider. Alternatively, you might want to test asking both for a quote and let customers compate. Whatever is the change you want to test, you will need to decide which API call to trigger before the customer is guaranteed to see pricing because you have to wait for confirmation, say, that the items are in stock.
+
+In this case too, the assignment happens when confirming basket contents and the address, but the entry point should be set to a little later, when the customer sees a quote.
+
+::: note
+
+If one of the API integration fails, and some customers assigned to Treatment don’t see the pricing quote because of a bug, then you’ll observe an [asymetric traffic](/statistics/sample-ratio-mismatch.md).
+
+:::
+
+There are more cases where tests on your deliveries would need an Entry point. Let’s say you want to improve the box opening experience, include a note, etc. The assignment has to hapen when packaging; the entry point has to be set when the customer receives the note. There’s no reason to include customers who didn’t get their packages when measuring the impact of a hand-written note on their satisfaction.
+
+Not all logistical test require a separate Entry point. If you want to test a promotion that offers free delivery under certain condition, you can decide which customer sees that promotion when it becomes visible, not earlier.
+
+::: note
+
+Be mindful when testing promotions on anonymous users: you risk contaminating your experiment. If they get re-assigned by refreshing their browser, or opening a private window, some users will stumble on it, notice the discrepancy, and figure out the trick. They might leak that information. If prospective buyers see it, they will try. You are better off testing promotions on entities that buyers can’t change easily: established user account, phone number, postal address.
+
+:::
+
+### Third example: AI-generated customer service
+
+Finally, let’s say you want to test one of those chat-bots that promise to replave human-lead customer service. The vendor recommends to send them as much information about the customer as you can, including their queries and the product decription that they open during their session. Small issue: user logs are streamed to your data warehouse, but you don’t really store that information in the browser memory or a convenient session object that could be sent it as single blob to the AI company. No problem: the vendor can ingest page content as the visitor goes through your site, store them and will use those as context when a customer asks a question.
```

Comment:
```suggestion
Finally, let’s say you want to test an AI chatbots for customer service. The vendor recommends to send them as much information about the customer as you can, including their queries and the product description that they open during their session. You accomplish this by sending the vendor page content as the visitor goes through your site, storing that information and using it as context when a customer asks a question.
```
A few sentences about implementation that aren't relevant and can be cut. You just need to establish that user data is sent to the AI bot before a user sees it.